gammapy/cube/make.py

```diff
@@ -28,7 +28,7 @@ class MapMaker:
         Exclusion mask
     """
 
-    def __init__(self, geom, offset_max, geom_true=None, exclusion_mask=None):
+    def __init__(self, geom, offset_max, geom_true=None, exclusion_mask=None, bg_coord_system="fov_altaz"):
```

Comment:
Would you mind renaming `bg_coord_system` to `bkg_coordsys`? Just because the abbreviations `bkg` and `coordsys` are more common in Gammapy.